src/main/kotlin/com/goldenraven/tatooine/TatooineWallet.kt

```diff
@@ -13,17 +13,16 @@ object TatooineWallet {
     private val electrumURL: String = "ssl://electrum.blockstream.info:60002"
     val log: Logger = LoggerFactory.getLogger(TatooineWallet::class.java)
 
+    private val feeRate:Float =1F;
```

Comment:
Try to use proper Kotlin syntax style: spaces between the variable name and the type, space after the assignment operator (`=`), and no semicolons at the end of lines.

This would result in the following line:
```kotlin
private val feeRate: Float = 1F
```